### ISSUE TITLE
Create About Section

### ISSUE THREAD

**Issue by Subhradeep10** [Subhradeep10/Fitnezz#4]

**Comment by Ritvik-UX** [comment]
About Page on website?
If yes, then, are we going to add another external CSS file to avoid confusions with main css file?

**Comment by Subhradeep10** [comment]
@Ritvik-UX Yes, you can add external Css, Better if you create another folder for about save the work there
